tf_seal/cc/ops/seal_ops.cc

```diff
@@ -72,3 +72,43 @@ REGISTER_OP("SealPolyEval")
     .Input("pub_key: variant")
     .Output("out: variant")
     .SetIsStateful();
+
+REGISTER_OP("SealSavePublickey")
+    .Input("filename: string")
+    .Input("pub_key: variant") 
+    .SetIsStateful();
+
+REGISTER_OP("SealSaveCipher")
```

Comment:
What is the difference between `SealSaveCipher` and `SealSaveCiphertext`?

Reply:
I have remove the second one.